server/@types/express/index.d.ts

```diff
@@ -27,7 +32,11 @@ export declare module 'express-session' {
       nonLinkedContacts: SocialContactsDto[]
     }
 
-    visitorRequest?: VisitorRequestForReviewDto
+    // Visitor request from booker for processing
+    visitorRequestJourney?: {
+      visitorRequest: VisitorRequestForReviewDto
+      linkedVisitors: VisitorInfoDto[]
+    }
```

Comment:
For visitor request processing journey, store request and already-linked visitors in session up-front to avoid needing to re-request.